virttest/utils_test/libvirt.py

```diff
@@ -3954,6 +3954,39 @@ def check_domuuid_compliant_with_rfc4122(dom_uuid_value):
     return dom_uuid_segments[2].startswith('4') and dom_uuid_segments[3][0] in '89ab'
 
 
+def check_xpaths(xml, xpaths, text=None):
```

Comment:
@smitterl  thank your efforts in bring this.
Just a open discussion:
1)Before avocado-vt PR:3615, VMXML.new_from_dumpxml will output device xml attributes in the order where attribute is set when creating device xml.
2)After avocado-vt PR:3615, VMXML.new_from_dumpxml will output device xml attributes in the order strictly complying with virsh command output : virsh dumpxml(I assumed it is from yesterday's email chain discussion)
3)In tp-libvirt, we don't have branches or releases out, so tp-libvirt just use master branch to be compatibility with avocado-vt master branch(that is our currently libvirt CI strategy). Therefore,if  the change happen in avocado-vt master, we just change tp-libvirt to match the change in avocado-vt master branch. That means specific to input devices issue this time, independent checking order may not be necessary.
4) I think this method can be useful , considering  future use cases